### PR TITLE
Change CODEOWNERS

### PR DESCRIPTION
**Pull request by alessioborraccino**
### :pushpin: References
* **Issue:** [SDK-XXX](https://gesundheitscloud.atlassian.net/browse/SDK-XXX)
* **Related pull-requests:** _list of related pull-requests (comma-separated): #1, #2 or **delete** this line_

### :see_no_evil: Parts covered

- [ ] Android
- [ ] Common
- [ ] iOS

### :thinking: DOD Checklist
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # General ownership
-* @gesundheitscloud/native-app-core
+* @gesundheitscloud/native-app-core @d4l-data4life/phdc-core
```
Comment (bitPogo) [inline .github/CODEOWNERS:2]: @wmontwe is `@gesundheitscloud/native-app-core` still needed?

Comment (wmontwe) [reply .github/CODEOWNERS:2]: It is not

Comment (bitPogo) [reply .github/CODEOWNERS:2]: @alessioborraccino Could please then remove it?

**Review by wmontwe — APPROVED** [review]
👍

**Pull request merged**